shared/requests/ledger_entry.yaml

```diff
@@ -37,8 +37,8 @@ components:
       description: The ledger_entry method returns a single ledger entry from the XRP Ledger in its raw format. All information retrieved is relative to a particular version of the ledger.
       type: object
       allOf:
-        - $ref: '../base.yaml#/components/schemas/LookupByLedgerRequest'
```

Comment:
How does the order of the schemas matter? Except probably inside the inheritance of classes?

Reply:
Yes it's for inheritance of classes. AccountInfo(BaseRequest, LookupByLedgerIndex) would make more sense than the other way around. Just a nit fix